Lib/test/test_httplib.py

```diff
@@ -1877,6 +1877,16 @@ def test_connect_with_tunnel(self):
         # This test should be removed when CONNECT gets the HTTP/1.1 blessing
```

Comment:
This comment is now redundant and can be removed, but I don't see a reason to remove the test itself.